crates/query-engine/metadata/src/metadata/mod.rs

```diff
@@ -11,7 +11,7 @@ use schemars::JsonSchema;
 use serde::{Deserialize, Serialize};
 
 /// Metadata information.
-#[derive(Debug, Default, Serialize, Deserialize, JsonSchema)]
+#[derive(Clone, Debug, Default, Serialize, Deserialize, JsonSchema)]
```

Comment:
I'm wondering where this is used. One nice property we had before was that we only create the metadata at startup and we reuse it.

Reply:
Yeah. That property has gone out the window with this change unfortunately, because `as_runtime_configuration` / `RawConfiguration` / `RuntimeConfiguration` no longer track lifetimes like they used to.

This is because I judged that tracking lifetimes would become more complex now that we have `RawConfiguration` being a union type, and with the whole manual serialization circus going on.

But once the PR is merged I think it would be an interesting thing to investigate separately to see if we can bring that property back. IIRC, when we were last looking at optimizing performance and looking at flamegraphs we _did_ spend a lot of relative effort cloning configurations.